Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -5121,22 +5121,36 @@ RefPtr<StyleValue> Parser::parse_filter_value_list_value(Vector<ComponentValue>
 RefPtr<StyleValue> Parser::parse_flex_value(Vector<ComponentValue> const& component_values)
 {
     if (component_values.size() == 1) {
+        // One-value syntax: <flex-grow> | <flex-basis> | none
         auto value = parse_css_value(component_values[0]);
         if (!value)
             return nullptr;
 
-        switch (value->to_identifier()) {
-        case ValueID::Auto: {
-            auto one = NumericStyleValue::create_integer(1);
-            return FlexStyleValue::create(one, one, IdentifierStyleValue::create(ValueID::Auto));
+        if (value->is_identifier() && property_accepts_value(PropertyID::Flex, *value)) {
```

Comment:
Hmmm. It's not your bug, but `property_accepts_value()` always returns `true` for shorthands, which makes that check redundant... that seems a bit silly.

Should be fine here since `flex-grow` and `flex-shrink` can't accept identifiers, but it's still a bit weird.